backend/route/views.py

```diff
@@ -128,6 +128,43 @@ def post_create(request):
     else:
         return HttpResponse(status=400)
 
+@require_POST
+def search(request):
+    try:
+        body = request.body.decode()
+        keyword = json.loads(body)['keyword']
+        location = json.loads(body)['location']
+        season = json.loads(body)['season']
+        days = json.loads(body)['days']
+        theme = json.loads(body)['theme']
+        transportation = json.loads(body)['transportation']
+    except (KeyError, JSONDecodeError):
+        return HttpResponseBadRequest()   
+    postlist=[]
+    for post in Post.objects.all():
+        place_exist=False
+        for place in post.place_set.all().order_by('day', 'index'):
+            if keyword!='' and keyword in place.description:
+                place_exist=True
+        if not place_exist:
+            if not(keyword!='' and (keyword in post.title or keyword in post.location)) or not(location!='' and location in post.location) or not (season!='' and season==post.season) or not (days!='' and days==post.days) or not (theme!='' and theme==post.theme) or not (transportation!='' and transportation==post.transportation):
+                continue
+        postlist.append({
```

Comment:
I think you don't need to send all the properties on response. Just like getting liked_posts or shared_posts on account, you can exclude detailed information because it is not showed on the card. Instead, you should include like_count and comment_count.

Here's the link:
https://github.com/swsnu/swpp2021-team4/blob/ce1d79c68b301ccc0c566a3df19b21c91068399d/backend/account/views.py#L263